forest_benchmarking/dfe.py

```diff
@@ -51,66 +41,128 @@ def calibrate_readout_imperfections(pauli: PauliTerm, quantum_machine: QuantumCo
     return expectation[0], variance[0, 0].real, count
 
 
-@dataclass
-class DFEexperiment:
+def _state_to_pauli(state: TensorProductState) -> PauliTerm:
+    term = sI()
+    for oneq_st in state.states:
+        if oneq_st.label == 'X':
+            term *= sX(oneq_st.qubit)
+        elif oneq_st.label == 'Y':
+            term *= sY(oneq_st.qubit)
+        elif oneq_st.label == 'Z':
+            term *= sZ(oneq_st.qubit)
+        else:
+            raise ValueError(f"Can't convert state {state} to a PauliTerm")
+
+        if oneq_st.index == 1:
```

Comment:
This is probably more a statement about the structure of `_OneQState` (in pyquil/operator_estimation) than the code here, but using `index` to denote the eigenstate can potentially be quite confusing: almost everywhere else in our code-base, we use index to denote the qubit on which the operation is occurring.

Reply:
we should move towards using "qubit" to denote qubits. Most of the gates use this convention

Reply:
Sounds good to me.